### DISCUSSION TITLE
V5 : any chance to support waves effect?

### DISCUSSION THREAD

**Discussion by devhoussam** [twbs/bootstrap#33362] [General]
http://fian.my.id/Waves/

**Comment by ascensus-mdb** [comment]
Here you go - Bootstrap 5 & waves effect integrated together

https://mdbootstrap.com/docs/standard/methods/ripple/

**Reply by devhoussam** [reply]
I know about **MDB** but what I mean is to support **waves effect** officially in **bootstrap 5** version.

I am not a fan of **Material Design**
